packages/spor-expandable-react-native/src/Expandable.tsx

```diff
@@ -0,0 +1,113 @@
+import { useState } from 'react';
+import {
+  composeRestyleFunctions,
+  createVariant,
+  SpacingProps,
+  spacingShorthand,
+  SpacingShorthandProps,
+  useRestyle,
+  useTheme,
+  VariantProps,
+} from "@shopify/restyle";
+import React from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { Theme } from "@vygruppen/spor-theme-react-native";
+import { Text } from '@vygruppen/spor-typography-react-native';
+import { Pressable } from 'react-native';
+import { ExpandableItem } from './ExpandableItem';
+import { DropdownDownFill18Icon, DropdownUpFill18Icon, DropdownUpFill24Icon, DropdownDownFill24Icon, DropdownDownFill30Icon, DropdownUpFill30Icon } from '@vygruppen/spor-icon-react-native';
+
+
+type RestyleProps = SpacingProps<Theme> &
+  SpacingShorthandProps<Theme> &
+  VariantProps<Theme, "expandableVariant", "variant">
+const variant = createVariant({ themeKey: "expandableVariant" });
+
+const restyleFunctions = composeRestyleFunctions<Theme, RestyleProps>([
+  spacingShorthand,
+  variant
+]);
+
+type ExpandableVariant = "card" | "outline" | "text";
+type ExpandableSize = "sm" | "md";
+
+type ExpandableProps = Exclude<RestyleProps, "variant"> & {
+  label: string,
+  children?: React.ReactNode,
+  leftIcon?: JSX.Element,
+  variant: ExpandableVariant,
+  isInitiallyExpanded?: Boolean
+  size?: ExpandableSize,
+  onToggleIsExpanded?: (isExpanding: boolean) => void
+};
+
+export const Expandable = ({
+  label,
+  leftIcon,
+  onToggleIsExpanded,
+  variant,
+  children,
+  size = "sm",
+  isInitiallyExpanded = false,
+  ...props
+}: ExpandableProps) => {
+  const theme = useTheme<Theme>();
+  const restyleProps: Record<string, any> = { ...props, variant };
+  const { style } = useRestyle(restyleFunctions, restyleProps);
+  const [isPressed, setIsPressed] = useState(false);
+  const pressedStyle = theme.getExpandableVariantPressedState(variant)
+  const [isExpanded, toggleExpanded] = useState(isInitiallyExpanded)
+  function handleToggleIsExpanded() {
+    toggleExpanded(!isExpanded)
+  }
+  function onPress(): void {
+    setIsPressed(true)
+    handleToggleIsExpanded()
+  }
+
+  return (
+    <Box style={style as any}>
+      < Pressable style={isPressed ? pressedStyle : { padding: 12 }}
+        onPressIn={() => { onPress() }}
```

Comment:
Her kan du gjøre én av to ting:

1. Bare sende inn `onPress` uten å kalle den (det gjør React for deg):
```suggestion
        onPressIn={onPress}
```
2. Inline onPress-funksjonen fra over, siden det tross alt ikke er så mye kode:
```suggestion
        onPressIn={() => { 
          setIsPressed(true);
          toggleExpanded(prev => !prev);
        }}
```